### DISCUSSION TITLE
Removing/Deleting users

### DISCUSSION THREAD

**Discussion by icsy7867** [uxbox/uxbox#2680] [General]
Testing out a container.  I know the manage.sh script helps to create users, but is there a clean/easy way to delete them as an administrator?

**Comment by niwinz** [comment]
Right now, no. :(  as a workaround you can access to the DB.
I'm personally working right now on an administration panel for penpot, for make the administration of onpremise setups more easily. I can't give you a deadline for it, but this is my current task this week :D

**Reply by icsy7867** [reply]
> Right now, no. :( as a workaround you can access to the DB. I'm personally working right now on an administration panel for penpot, for make the administration of onpremise setups more easily. I can't give you a deadline for it, but this is my current task this week :D

Neat! I appreciate the quick response.  World deleting the users from the user table be sufficient?

**Reply by niwinz** [reply]
It is better to mark them as deleted and leave the system to perform all the cleanup task (instead just deleting the)

```
update profile set deleted_at=now() where <your-conditions-here>;
```

The profile deletion just removes the profile and its relations with teams, but does not removes the teams, and all team content (project and files); so just removing profiles will leave probably unreachable objects on the db. I'm also thinking on how to improve this part as part of my task :D

**Reply by icsy7867** [reply]
That's really easy!  I'll probably just write my own shell script.  Thank you!
